README.md

```diff
@@ -409,7 +409,8 @@ Built-in functions. Ready to be bound to any key you like. :smile:
 | `builtin.lsp_workspace_symbols`     | Searches in LSP all workspace symbols.                                                      |
 | `builtin.lsp_code_actions`          | Lists LSP action to be trigged on enter.                                                    |
 | `builtin.lsp_range_code_actions`    | Lists LSP range code action to be trigged on enter.                                         |
-| `builtin.lsp_diagnostics`           | Lists LSP Diagnostics in the current document.                                              |
+| `builtin.lsp_buffer_diagnostics`    | Lists LSP Diagnostics in the current document.                                              |
```

Comment:
Maybe it's just me being a little bit too nitpicky, but I think this is better because it matches with the description :)

```suggestion
| `builtin.lsp_document_diagnostics`    | Lists LSP Diagnostics in the current document.                                            |
```

Reply:
Fully agree! Too hastily on my end :)